master/static/migrations/20240828101021_custom-webhook-trigger.tx.up.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TYPE trigger_type RENAME TO _trigger_type;
+
+CREATE TYPE trigger_type AS ENUM (
+  'EXPERIMENT_STATE_CHANGE',
+  'METRIC_THRESHOLD_EXCEEDED',
+  'TASK_LOG',
```

Comment:
Can't we extend the current `trigger_type` enum to include `CUSTOM`? Or does postgres not allow adding to enums?

Reply:
Gotcha, makes sense!